src/components/common/FamilyIntake.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Form, Input, Button, Select, InputNumber } from 'antd';
+
+function FamilyIntake() {
+  const { TextArea } = Input;
+
+  // Initial Empty Values for form
+  const familyInput = {
+    head_of_household: '',
+    family_members: '',
+    family_size: null,
+    education_level: '',
+    homeless_history: '',
+    employment: '',
+  };
+
+  const [familyInfo, setFamilyInfo] = useState(familyInput);
+
+  // Change Handler used for all fields except Select.
+  const formChanges = event => {
+    const { name, value } = event.target;
+    setFamilyInfo({ ...familyInfo, [name]: value });
+  };
+  // antd's answer to onSubmit
+  const onFinish = () => {
+    console.log(familyInfo);
+  };
+
+  return (
+    <div>
```

Comment:
Is this div necessary?

Reply:
No, it is not. <Form> can function as the most parent element.